src/components/Message/styles.css

```diff
@@ -0,0 +1,32 @@
+.message {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    padding: var(--dui-spacing-medium);
```

Comment:
use an appropriate variable. We need to replace`--dui--` variable with the corresponding `--tui--` variable